sky/clouds/oci.py

```diff
@@ -358,6 +365,13 @@ def check_credentials(cls) -> Tuple[bool, Optional[str]]:
             del user
             # TODO[Hysun]: More privilege check can be added
             return True, None
+        except (oci_adaptor.get_oci().exceptions.ConfigFileNotFound,
+                oci_adaptor.get_oci().exceptions.InvalidConfig) as e:
+            # This should only happen in testing where oci config is
+            # monkeypatched.
```

Comment:
Is it possible that in non-testing environment, the user can have some invalid configuration that reaches here? If not, might worth commenting about why.